docs/walrus-sites/tutorial-install.md

```diff
@@ -8,30 +8,46 @@ prepare your environment for development.
 Before you start, make sure you
 
 - have a recent version of [Rust](https://www.rust-lang.org/tools/install) installed;
-- have `git` installed; and
 - followed all [Walrus setup instructions](../usage/setup.md).
 
 Then, follow these additional setup steps.
 
-## Clone the repository and build the `site-builder` tool
+## Installation
 
-First, clone and enter the Walrus Sites repo from <https://github.com/MystenLabs/walrus-sites>:
+Similar to the `walrus` client cli tool, we currently provide the `site-builder`
+client binary for macOS (Intel and Apple CPUs), Ubuntu, and Windows:
 
-``` sh
-git clone https://github.com/MystenLabs/walrus-sites.git
-cd walrus-sites
+| OS      | CPU                   | Architecture                                                                                                                 |
+| ------- | --------------------- | ---------------------------------------------------------------------------------------------------------------------------- |
+| Ubuntu  | Intel 64bit           | [`ubuntu-x86_64`](https://storage.googleapis.com/mysten-walrus-binaries/site-builder-testnet-latest-ubuntu-x86_64)                 |
+| Ubuntu  | Intel 64bit (generic) | [`ubuntu-x86_64-generic`](https://storage.googleapis.com/mysten-walrus-binaries/site-builder-testnet-latest-ubuntu-x86_64-generic) |
+| MacOS   | Apple Silicon         | [`macos-arm64`](https://storage.googleapis.com/mysten-walrus-binaries/site-builder-testnet-latest-macos-arm64)                     |
+| MacOS   | Intel 64bit           | [`macos-x86_64`](https://storage.googleapis.com/mysten-walrus-binaries/site-builder-testnet-latest-macos-x86_64)                   |
+| Windows | Intel 64bit           | [`windows-x86_64.exe`](https://storage.googleapis.com/mysten-walrus-binaries/site-builder-testnet-latest-windows-x86_64.exe)       |
+
+```admonish title="Windows"
+We now offer a pre-built binary also for Windows. However, most of the remaining instructions assume
+a UNIX-based system for the directory structure, commands, etc. If you use Windows, you may need to
+adapt most of those.
 ```
 
-Then, build the release version of the site builder:
+You can download the latest build from our Google Cloud Storage (GCS) bucket (correctly setting the
+`$SYSTEM` variable):
 
-``` sh
-cargo build --release
+```sh
+SYSTEM= # set this to your system: ubuntu-x86_64, ubuntu-x86_64-generic, macos-x86_64, macos-arm64, windows-x86_64.exe
+curl https://storage.googleapis.com/mysten-walrus-binaries/site-builder-testnet-latest-$SYSTEM -o site-builder
+chmod +x site-builder
 ```
 
-After the build process completes, you are ready to run the site builder:
+To be able to run it simply as `site-builder`, move the binary to any directory included
+in your `$PATH` environment variable. Standard locations are `/usr/local/bin/`, `$HOME/bin/`,
+or `$HOME/.local/bin/`.
+
+Once this is done, you should be able to simply type `site-builder` in your terminal.
```

Comment:
Suggestion: mention that this requires also putting the configuration in one of the standard locations or using `--config`